staging/src/k8s.io/apiserver/pkg/authentication/token/cache/cached_token_authenticator.go

```diff
@@ -201,10 +201,10 @@ func (a *cachedTokenAuthenticator) doAuthenticateToken(ctx context.Context, toke
 		ac := audit.AuditContextFrom(ctx)
 		// since this is shared work between multiple requests, we have no way of knowing if any
 		// particular request supports audit annotations.  thus we always attempt to record them.
-		ac.Event.Level = auditinternal.LevelMetadata
+		ac.SetEventLevel(auditinternal.LevelMetadata)
```

Comment:
When the event level is unset, the audit context records annotations (https://github.com/kubernetes/kubernetes/blob/42634e39d385b0dda8a96bc1be8807df94b2fbd2/staging/src/k8s.io/apiserver/pkg/audit/context.go#L51-L52), so I think we can actually remove this line, which also lets us remove `SetEventLevel` (although it's used a lot in tests)

Reply:
@tallclair will do it in a follow up PR. hope that's ok?